cloudigrade/api/authentication.py

```diff
@@ -32,6 +32,10 @@ def parse_requests_header(request):
     )
 
     auth_header = request.META.get(settings.INSIGHTS_IDENTITY_HEADER, None)
+    if self.allow_internal_fake_identity_header:
```

Comment:
Do we _always_ want to override the standard header if we're allowing the internal fake identity header? This is where my knowledge of the internal Turnike use case is a bit thin. Does Turnpike send anything at all in the position of the regular identity header? Or is it safe to just discard it entirely in favor of our fake one?

Reply:
Yes, this is fine. the only 2 things we care about are the account_number and is_org_admin which do not exist in the Turnpike generated header anyways.  When we communicate internally to sources we generate the identity header ourselves using the account number we obtained, in other words we never forward the whole header as is downstream so we really don't use any other info from the header.